CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Lotus changelog
 
+# v1.23.2 / 2023-06-28
+
+This is a patch release on top of 1.23.2 containing the fix for https://github.com/filecoin-project/lotus/issues/10906
```

Comment:
I would add more detail here:
- what was the issue
- who do we recommend upgrade (and with what urgency)